sky/optimizer.py

```diff
@@ -21,21 +21,20 @@
 from sky.utils import log_utils
 
 if typing.TYPE_CHECKING:
-    from sky import dag as dag_lib
+    import networkx as nx
+    from sky import dag as dag_lib  # pylint: disable=ungrouped-imports
 
 logger = sky_logging.init_logger(__name__)
 
-Task = task_lib.Task
```

Comment:
mypy (and format.sh) appears to pass with this.. any reason why we remove this line and use `task_lib.Task` everywhere in this file?

Reply:
Oh, since the `task_lib.Task` is not that long, I just replaced it to get rid of unnecessary abbrieviation. Do you think it looks better before?

Reply:
I think it was slightly cleaner to use `Task = task_lib.Task` since `task_lib.Task` gets tedious to read everywhere in the source file : ) I don't feel too strongly about it though, just a preference.